FtpClientApp/FtpClientApp/ListFiles.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.IO;
+using System.Net;
+
+namespace FtpClientApp
+{
+    class ListFiles
+    {
+        public ListFiles(ServerConnectionInformation myConnection)
+        {   // defaults used for testing
+            //myConnection.ServerName = "ftp://speedtest.tele2.net";
+            //myConnection.UserName = "anonymous";
+            //myConnection.PassWord = "anonymous";

```

Comment:
also it might be good to ask the user here if they want to list the files in a specific directory (with a blank response indicating to just list root as here)